### PR TITLE
CR-1123097 ASTeR basic sanity - vck5000 - VMR version not displayed in xbutil/xbmgmt examine

### PR DESCRIPTION
**Pull request by AShivangi**
#### Problem solved by the commit
Add vmr version support to reporting

#### How problem was solved, alternative solutions (if any) and why they were rejected
A lot of these flags are for debugging so I decided to hide them under verbose. Unless `--verbose` is specified, only the vmr version will be displayed. 
Subject to change based on ux

#### Risks (if any) associated the changes in the commit
None

#### What has been tested and how, request additional testing if necessary
Locally
Non verbose:
```
# xbmgmt examine -d -r vmr

-------------------------------------------------------
1/1 [0000:09:00.0] : xilinx_vck5000_gen4x8_xdma_base_2
-------------------------------------------------------
Vmr Status
  Build flags          :  default full build
  Vitis version        :  2022.1
  Git hash             :  390ee9310e6fb948077d7497f6bb71b3ebe8516d
  Git branch           :  apu
  Git hash date        :  Mon, 21 Mar 2022 11
```
Verbose:
```
# xbmgmt examine -d -r vmr --verbose
Verbose: Enabling Verbosity
Verbose: SubCommand: examine

-------------------------------------------------------
1/1 [0000:09:00.0] : xilinx_vck5000_gen4x8_xdma_base_2
-------------------------------------------------------
Vmr Status
  Build flags          :  default full build
  Vitis version        :  2022.1
  Git hash             :  390ee9310e6fb948077d7497f6bb71b3ebe8516d
  Git branch           :  apu
  Git hash date        :  Mon, 21 Mar 2022 11
  Is ps ready          :  1
  Default image offset :  0x48000
  Default image size   :  0x6fec60
  Default image capacity :  0x5fb8000
  Backup image offset  :  0x6008000
  Backup image size    :  0x6fec60
  Backup image capacity :  0x5fb8000
  Sc firmware size     :  0x630f3
  Has fpt              : 1
  Has fpt recovery     : 1
  Boot on default      : 1
  Boot on backup       : 0
  Boot on recovery     : 0
  Multi boot offset    : 0x9
  Has extfpt           : 1
  Has ext meta xsabin  : 1
  Has ext sc fw        : 1
  Has ext system dtb   : 1
  Debug level          : 0
  Program progress     : 100
  Pl is ready          : 1
  Ps is ready          : 1
```

### CODE REVIEW

**Review by xdavidz — APPROVED** [review]
Looks good to me.

**Review by stsoe — APPROVED** [review]

**Comment by dayeh-xilinx** [comment]
retest this please.

File: src/runtime_src/core/common/info_vmr.cpp [hunk 1]
```diff
@@ -29,6 +29,7 @@ static std::string
 pretty_label(std::string label)
 {
   std::replace(label.begin(), label.end(), '_', ' ');
+  label[0] = static_cast<char>(toupper(label[0]));
```
Comment (rozumx) [inline src/runtime_src/core/common/info_vmr.cpp:32]: What happens if the label is empty?

Comment (rozumx) [inline src/runtime_src/core/common/info_vmr.cpp:32]:
```suggestion
  if (!label.empty())
     label[0] = static_cast<char>(toupper(label[0]));
```

File: src/runtime_src/core/tools/common/ReportVmrStatus.cpp [hunk 2]
```diff
@@ -51,10 +55,20 @@ ReportVmrStatus::writeReport( const xrt_core::device* /*_pDevice*/,
     output << "  Information Unavailable" << std::endl;
     return;
   }
+
+  //list of non verbose labels
+  const std::vector<std::string> non_verbose_labels = { "build flags", "vitis version",
```
Comment (rozumx) [inline src/runtime_src/core/tools/common/ReportVmrStatus.cpp:60]: You might wish to reformat this list.  Maybe 1 item per line.

**Review by rozumx — CHANGES_REQUESTED** [review]

**Review by rozumx — APPROVED** [review]

**Pull request merged**